### ISSUE TITLE
Install fails

### ISSUE THREAD

**Issue by deljones** [YunoHost-Apps/invoiceninja5_ynh#19]
### Describe the bug

Sticks on install

*A clear and concise description of what the bug is.*

Installing IN5 into its own domain using the YNH admin interface.

Setup stops at “setting up source files

### Context

- Hardware: *VPS bought online / Old laptop or computer / Raspberry Pi at home / Internet Cube with VPN / Other ARM board / ...*
- YunoHost version: 4.3.6.3
- I have access to my server: *Through SSH | through the webadmin | direct access via keyboard / screen | ...*
- Are you in a special context or did you perform some particular tweaking on your YunoHost instance?: *no
  - If yes, please explain:

### Steps to reproduce

- Using the YNH admin page
- Applications
- Choose the domain to install to
- Select the admin account for the application
- Choose the language 
- Enter a password

Install begins…..

No change once “Setting up source files” is reached

Left like this for a good hour.( just in case) 

Go to the installation website. 404 error

### Expected behavior

Go to the domain and be presented with IN5 login details request to complete the install.

### Logs

Don’t know where to look?

*If applicable and useful, add screenshots to help explain your problem.*

**Comment by rndmh3ro** [comment]
I just tried it and for me it worked.
Can you please try again?

If it still doesn't work, try installing it like this from the command line:
```
sudo yunohost app install https://github.com/YunoHost-Apps/invoiceninja5_ynh/  --debug
```

Then report back what the output says.

**Comment by deljones** [comment]
@rndmh3ro Sorry for the late reply. I got this working.

The problem I had was that even though I had an empty domain, I had no users on the domain. I was trying to use the user from another one of my YNH domains as admin. Once I set up a user on the domain I wanted to use it installed fine, the migration from my v4 to v5 was very smooth and I'm back in business. Thanks for your work on this, it's excellent.

The only issue I have, and I have created a new topic with the Invoice Ninja forum, is that I can preview my expenses, ie upload a document or a bill, but if I try to preview it or download it I get a 404 error. This is still ongoing.

Also regarding the update. There is one available which won't happen. I get an error that says ".env is not writable". I assume that any updates to this package have to be done through the YNH web UI?

Thanks again for your help

Regards

dj
